src/components/sbb-wagon/sbb-wagon.scss

```
@@ -0,0 +1,88 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  --sbb-wagon-width: var(--sbb-train-formation-wagon-width);
+  --sbb-wagon-height: var(--sbb-train-formation-wagon-height);
+  --sbb-wagon-inner-padding: var(--sbb-spacing-fixed-3x);
+  --sbb-wagon-shape-border-radius: var(--sbb-train-formation-wagon-border-radius);
+  --sbb-wagon-shape-outline-color: var(--sbb-color-charcoal-default);
+  --sbb-wagon-shape-outline-color-block: var(--sbb-color-metal-default);
+  --sbb-wagon-shape-outline-width: var(--sbb-border-width-1x);
+  --sbb-wagon-icons-padding-block-start: var(--sbb-spacing-fixed-2x);
+  --sbb-wagon-icons-gap: var(--sbb-spacing-fixed-1x);
+}
+
+.sbb-wagon {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: var(--sbb-wagon-width);
+
+  :host([type='wagon']) &,
+  :host([type='closed']) & {
+    width: var(--sbb-wagon-width);
+  }
+}
+
+.sbb-wagon__label {
+  @include sbb.text-s--regular;
+}
+
+.sbb-wagon__label-screenreader {
+  @include sbb.screen-reader-only;
+}
+
+.sbb-wagon__label-text {
+  display: inline-block;
+}
+
+.sbb-wagon__compartment {
+  height: var(--sbb-wagon-height);
+
+  :host([type='wagon']) &,
+  :host([type='closed']) & {
+    padding: 0 var(--sbb-wagon-inner-padding);
+    display: flex;
+    align-items: center;
+    width: 100%;
+    border-radius: var(--sbb-wagon-shape-border-radius);
+  }
+
+  :host([type='wagon']) & {
+    border: var(--sbb-wagon-shape-outline-width) solid var(--sbb-wagon-shape-outline-color);
+  }
+
+  :host([type='closed']) & {
+    border: var(--sbb-wagon-shape-outline-width) solid var(--sbb-wagon-shape-outline-color-block);
```

Should the outline var be called closed instead of block?
```suggestion
    border: var(--sbb-wagon-shape-outline-width) solid var(--sbb-wagon-shape-outline-color-closed);
```